app/views/logs/index.html.erb

```diff
@@ -1,29 +1,30 @@
+<%= link_to "Back to search", search_logs_path, class: "govuk-back-link" %>
+
 <% if @logs.empty? %>
-  <h3>No results found for that username, please try again</h3>
+  <h3 class="govuk-heading-m">No results found for that username, please try again</h3>
 <% else %>
-
-  <h1>Displaying logs for <%= @logs.first.fetch("username") %></h1>
+  <h1 class="govuk-heading-l">Displaying logs for <%= @logs.first.fetch("username") %></h1>
   <table class="govuk-table">
     <caption class="govuk-table__caption"><%= @logs.count %> results</caption>
     <thead class="govuk-table__head">
       <tr class="govuk-table__row">
         <th class="govuk-table__header" scope="col">Username</th>
-        <th class="govuk-table__header" scope="col">AP</th>
-        <th class="govuk-table__header" scope="col">MAC</th>
-        <th class="govuk-table__header" scope="col">siteIP</th>
-        <th class="govuk-table__header" scope="col">Start</th>
-        <th class="govuk-table__header" scope="col">Success</th>
+        <th class="govuk-table__header" scope="col">Access Point</th>
+        <th class="govuk-table__header" scope="col">MAC Address</th>
+        <th class="govuk-table__header" scope="col">IP</th>
+        <th class="govuk-table__header" scope="col">Time</th>
+        <th class="govuk-table__header" scope="col">Status</th>
```

Comment:
I like `Result` better than `Status` - the latter implies that this is something still in-progress, could be confusing?